packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -168,6 +168,18 @@ let didWarnSetStateChildContext;
 let warnAboutUpdateOnUnmounted;
 let warnAboutInvalidUpdates;
 
+if (enableSchedulerTracking) {
+  // Provide explicit error message when production+profiling bundle of e.g. react-dom
+  // Is used with production (non-profiling) bundle of schedule/tracking
```

Comment:
Nit: does this line need to start with capital letter? Made reading the sentence a bit confusing.

Reply:
No, I can change the comment 😆